bundler.d/dhcp_isc.rb

```diff
@@ -7,3 +7,8 @@
 group :dhcp_isc_kqueue do
   gem 'rb-kqueue'
 end
+
+# these gems were moved out of the default ruby gems in >=3.4
+if RUBY_VERSION >= '3.4'
+  gem 'observer'
```

Comment:
I can't find where we use the `observer` gem. https://github.com/ruby/observer says you use it with `include Observable` but AFAIK dhcp_isc only depends on inotify or kqueue to get file updates.

Reply:
Ah, it's [rubyipmi](https://github.com/logicminds/rubyipmi) that actually needs it. I've opened https://github.com/logicminds/rubyipmi/pull/60.